mobile/mysterium/openvpn_connection_setup.go

```diff
@@ -65,3 +85,45 @@ func (adapter channelToCallbacksAdapter) OnStats(openvpnStats openvpn3.Statistic
 		BytesReceived: uint64(openvpnStats.BytesIn),
 	})
 }
+
+// Openvpn3TunnelSetup is alias for openvpn3 tunnel setup interface exposed to Android/iOS interop
+type Openvpn3TunnelSetup openvpn3.TunnelSetup
+
+// OverrideOpenvpnConnection replaces default openvpn connection factory with mobile related one
+func (mobNode *MobileNode) OverrideOpenvpnConnection(tunnelSetup Openvpn3TunnelSetup) {
```

Comment:
Maybe we can call it `MobileOpenvpnConnection`. From user standpoint he is not overriding anything..

Reply:
See my comment above